### ISSUE TITLE
[feature request] 

### ISSUE THREAD

**Issue by BrainOppotamus** [gnembon/carpet-extra#137]
I would love a carpet (extra) rule that reintroduce Redstone wire loading chunks for itself. (just the basic 1.12 dustbehavior when it comes to Redstone signals going further then render distance)

**Comment by Megaminx12** [comment]
It might be a good idea to edit 'Redstone Loading Chunks like 1.12' or smth into the title.

**Comment by Copqer** [comment]
thats a very good idea but would lag the shit out of carpet servers lol

**Comment by DragonEggBedrockBreaking** [comment]
> 
> 
> thats a very good idea but would lag the shit out of carpet servers lol

not necessarily; remember that all carpet rules are disabled by default, so someone with adequate permission would have to explicitly enable this rule for it to "lag the shit out of" the server
